### ISSUE TITLE
API that require package root

### ISSUE THREAD

**Issue by GMartigny** [vercel/vercel#3700]
Hi,

I try to deploy an API using the the package of a repo. Basically, my working directory look like:
 - api/
	- index.js
 - index.js
 - package.json

Inside `./index.js`, I export a function that I publish on `NPM`.
Inside `./api/index.js`, I want to make use of that function, but I don't know how.

If I do:
```js
const fn = require("..");
```
it works on my machine, but not once deployed to `now` because the package files are not kept.

I can't do:
```js
const fn = require("my-package");
```
because `"my-package"` is not found on `node_modules`. Of course, it's not possible to install `"my-package"` under itself.

I tried to add a `package.json` under the `api` directory. It gives me an error that I don't understand:

	Error: libuuid.so.1: cannot open shared object file: No such file or directory

Do I have to create a new project only for the API deploy ? This doesn't look right.

**Comment by styfle** [comment]
Hi @GMartigny 

The `require('..')` usage was fixed in https://github.com/zeit/node-file-trace/pull/92 and shipped in #3613.

Can you confirm that this is working?

**Comment by GMartigny** [comment]
Unfortunately, I cannot confirm that it fix my issue.

You can see the [reproducing repo branch](https://github.com/GMartigny/lorem-pix/tree/nowDotDotRequire) and [the associated deployment](https://lorem-pix-aif3bwcen.now.sh/api/index) showing the same issue.

![image](https://user-images.githubusercontent.com/2543511/76963882-090ef080-6922-11ea-9f41-6cbd579f6153.png)

> Error: libuuid.so.1: cannot open shared object file: No such file or directory

I also have the same issue with `require("../..")` which is not covered by the mentioned fix.

**Comment by styfle** [comment]
I think the `libuuid.so.1` issue is unrelated to the required path. This is likely because you have a native dependency that is dynamically linked and expects that file to be available on the host even though it is not.

You could try changing `const fn = require("..");` to `const fn = require("../index.js");` to see that the behavior should be the same.

**Comment by GMartigny** [comment]
> I think the `libuuid.so.1` issue is unrelated to the required path. This is likely because you have a native dependency that is dynamically linked and expects that file to be available on the host even though it is not.

You're right. It seems to have a compatibility issue with [`canvas`](https://github.com/Automattic/node-canvas) current version (`2.6.1`). I'm sure it used to work on `2.4.1`.
Sadly, the [working deployment](https://lorem-pix-o5fjmzam5.now.sh) got removed (certainly because I'm a non paying user).

This issue is not about the `require`'s path as I thought. Should I raised a new one ?
